test-helpers/platform-test-helpers.js

```diff
@@ -1,7 +1,7 @@
 // This file is shared with voyager-api/voyager-publish
 // Helpers in this file must not have anything project-specific like measurement fields
 
-import { Point } from '../publish/lib/telemetry.js'
+import { Point } from '../common/telemetry.js'
```

Comment:
Perhaps we should import the Point directly from InfluxDB SDK to avoid calling the code that initialises the writers?

Reply:
Addressed in https://github.com/filecoin-station/spark-api/pull/339/commits/6332977b75dd30bd098b3c751b5efbefb59a2d0e